### PR TITLE
Login and forgot password

### PR DESCRIPTION
**Pull request by veiga91**

### CODE REVIEW

File: packages/app/client/src/App.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ import React from 'react'
 import { Provider } from 'react-redux'
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
 import { logoutUser, setCurrentUser } from './actions/authActions'
-import Login from './components/auth/Login'
-import ForgotPassword from './components/auth/ForgotPassword'
+//import Login from './components/auth/Login'
+//import ForgotPassword from './components/auth/ForgotPassword'
```
Comment (campbell-codes) [inline packages/app/client/src/App.js]: Delete commented out code

File: packages/app/client/src/api/users.js [hunk 2]
```diff
@@ -28,8 +39,26 @@ export const sendEmailToSignUp = userData => {
...
+    if (err.response && err.response.status) {
+      switch (err.response.status) {
+        case 403:
+          errorMessage = ERRORS.AWAITING_VERIFICATION
+          break
+        case 404:
+          errorMessage = ERRORS.EMAIL_NOT_FOUND
+          break
+        default:
+          break
```
Comment (campbell-codes) [inline packages/app/client/src/api/users.js]: This is nice how it's done

File: packages/app/client/src/ui/pages/ForgotPassword.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  const [step, changeStep] = useState(2)
+  const containerStyle = useContainerStyle({ size: 'small' })
+
+  const verifyEmail = async ({ email }) => {
+    try {
+      await sendForgotPasswordEmail({ email })
+      changeStep(1)
+    } catch(e) {
+      // TODO: add handler
+      console.log(e)
```
Comment (campbell-codes) [inline packages/app/client/src/ui/pages/ForgotPassword.js]: This should be implemented now no?

File: packages/app/client/src/ui/pages/ForgotPassword.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  }
+  const showSuccess = step === 2
+
+  const saveNewPassword = async ({ password }) => {
+    try {
+      await changeUserPassword({ password })
+      changeStep(2)
+    } catch(e) {
+      // TODO: add handler
+      console.log(e)
```
Comment (campbell-codes) [inline packages/app/client/src/ui/pages/ForgotPassword.js]: Same as above

File: packages/app/client/src/ui/pages/Login.js [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+const Login = () => {
+  const sendLoginRequest = async ({ email, password }) => {
+    try {
+      await loginUser({ email, password })
+    } catch(e) {
+      // TODO: add handler
+      console.log(e)
+    }
```
Comment (campbell-codes) [inline packages/app/client/src/ui/pages/Login.js]: Maybe for these we can just do a simple "Something went wrong" error for now

**Review by campbell-codes — CHANGES_REQUESTED** [review]
Generally looks good a few comments

File: packages/app/client/src/ui/pages/ForgotPassword.js (hunk 3 above)
Comment (veiga91) [reply packages/app/client/src/ui/pages/ForgotPassword.js]: I implemented something to deal with it in this PR -> https://github.com/unicef/atrium/pull/6

Comment (veiga91) [reply packages/app/client/src/ui/pages/ForgotPassword.js]: I don’t If you meant a handler like get the error code and throw a message or show something to the user

Comment (veiga91) [reply packages/app/client/src/ui/pages/ForgotPassword.js]: I am on this. I will change that from the code perspective to handle the error and show it to the user in a better way

**Review by campbell-codes — APPROVED** [review]

**Pull request merged**